src/languages/markdown.js

```diff
@@ -71,16 +76,19 @@ export default function(hljs) {
       },
       {
         className: 'link',
+        relevance: 0,
```

Comment:
The relevance here all happens in the top LINK rule, not in the children.  Relevance here is counted a single time, not layered.